eng/Packages.Data.props

```diff
@@ -295,6 +295,7 @@
     <PackageReference Update="Microsoft.ApplicationInsights" Version="2.20.0" />
     <PackageReference Update="Microsoft.Azure.ApplicationInsights.Query" Version="1.0.0" />
     <PackageReference Update="Microsoft.AspNetCore" Version="2.2.0" />
+    <PackageReference Update="Microsoft.AspNetCore.Mvc.Core" Version="2.2.0" />
```

Comment:
These packages are legacy and no longer maintained.   Is there a way to not use them so that we don't trigger additional component governance alerts that we have to add ignore rules for?


- https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc
- https://www.nuget.org/packages/Microsoft.AspNetCore

Reply:
I removed the dependency for now. I will have to figure out what to do here longer term though. These packages seem to be obsolete without an alternative with these attributes.